dev/scripts/components/Emblem.js

```diff
@@ -1,15 +1,18 @@
 import React from 'react';
 
-const Emblem = ({ type }) => {
-  const capitalType = type.charAt(0).toUpperCase() + type.slice(1);
+const Emblem = (props) => {
+  // const capitalType = type.charAt(0).toUpperCase() + type.slice(1);
 
   return (
     <div className="typeHolder">
-      <img
-        src={`../../../images/${capitalType}.png`}
-        alt={`an emblem of the type ${capitalType}`}
-        title={`${capitalType} Type Icon`}
-      />
+      <label htmlFor={`${props.type}`}>
```

Comment:
Would "value={${props.type}}" be equivalent to value={props.type} ?
Same applies to id.